__tests__/jesttest.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import * as TestUtils from 'react-dom/lib/ReactTestUtils';
+import {shallow} from 'enzyme';
+import {expect} from 'chai';
+import * as LibraryEntryPoint from '../src/entry-point';
+import LibraryContainer from '../src/components/LibraryContainer';
+import LibraryItem from '../src/components/LibraryItem';
```

Comment:
Hi @fanwgwg, we should update `ReadMe.md` to include new information of how tests can be run.